doc/onboarding-extras.md

```diff
@@ -13,6 +13,7 @@
 | `lib/{crypto,tls,https}` | @nodejs/crypto |
 | `lib/domains` | @misterdjules |
 | `lib/fs`, `src/{fs|file}` | @nodejs/fs |
+| `lib/internal/url` | @nodejs/url |
```

Comment:
Should it be `lib/url.js`? `internal` seems to be implied everywhere else (e.g., `lib/fs` would presumably include `lib/internal/fs.js`).